fotoapparat/src/main/java/io/fotoapparat/routine/zoom/UpdateZoomLevelRoutine.kt

```diff
@@ -17,7 +18,7 @@ internal fun Device.updateZoomLevel(
     zoomLevel.ensureInBounds()
     val cameraDevice = awaitSelectedCamera()
 
-    if (cameraDevice.getCapabilities().canZoom) {
+    if (cameraDevice.getCapabilities().zoom is Zoom.VariableZoom) {
```

Comment:
java users will hate us for this but 🤷‍♂️

Reply:
Hmm, maybe will be better if I return "canZoom" parameter?...

Reply:
Let's leave it as is. We focus on Kotlin first after all